add-chain/main.go

```diff
@@ -92,60 +73,33 @@ func entrypoint(ctx *cli.Context) error {
 		return fmt.Errorf("failed to get superchain level: %w", err)
 	}
 
+	publicRPC := os.Getenv("SCR_PUBLIC_RPC")
+	sequencerRPC := os.Getenv("SCR_SEQUENCER_RPC")
+	explorer := os.Getenv("SCR_EXPLORER")
+	superchainTarget := os.Getenv("SCR_SUPERCHAIN_TARGET")
+	monorepoDir := os.Getenv("SCR_MONOREPO_DIR")
```

Comment:
Any reason we can't promote these to flags like the other variables? That way, we can still read them in from the environment (even the .env files), but also have the option to read in via command line flags. And it would simplify / unify the code a bit.

Reply:
https://github.com/ethereum-optimism/superchain-registry/pull/349/commits/1ec8c9ead5f51af318146b9665bfab388185fee3